src/pages/organizers/create/OrganizerForm.tsx

```diff
@@ -72,6 +99,20 @@ const OrganizerForm = () => {
           form={form}
         />
       </Page.Content>
+      <Page.Footer>
+        <Button
+          variant={ButtonVariants.SUCCESS}
+          onClick={async () =>
+            await createOrganizer.mutateAsync({
+              name: form.getValues().nameAndUrl.name,
+              url: form.getValues().nameAndUrl.url,
+            })
```

Comment:
mainLanguage is mandatory
```suggestion
            await createOrganizer.mutateAsync({
              name: form.getValues().nameAndUrl.name,
              url: form.getValues().nameAndUrl.url,
              mainLanguage: i18n.language
            })
```